src/components/SearchResult/SearchResult.js

```diff
@@ -183,15 +193,15 @@ class SearchResult extends Component {
           </Header>
           <Image
             style={{ display: 'block', margin: '0 auto' }}
-            src="https://media2.giphy.com/media/y1ZBcOGOOtlpC/200.gif"
+            src={loadingIcon}
           />
         </Container>
       );
     }
     return (
       <Container>
         <Header as="h1" style={{ padding: '20px' }}>
```

Comment:
try to avoid using inline styles, use classNames

Reply:
done.